src/www/electron.webpack.js

```diff
@@ -54,7 +54,7 @@ module.exports = makeConfig({
     // @sentry/electron depends on electron code, even though it's never activated
     // in the browser. Webpack still tries to build it, but fails with missing APIs.
     // The IgnorePlugin prevents the compilation of the electron dependency.
-    new webpack.IgnorePlugin(/^electron$/),
+    new webpack.IgnorePlugin({resourceRegExp: /^electron$/}),
```

Comment:
Why is this changing? Why is this needed?

Reply:
i'm not 100% sure 🙁 it was failing here https://github.com/Jigsaw-Code/outline-client/runs/4121937811?check_suite_focus=true#step:7:26 the version must be changed? but i grepped the `package-lock` and didn't see anything obvious. and this passed on the previous PR so... I'm a little confused as well

Reply:
it must be this minor upgrade? Or one of the adjacent ones?

![Screen Shot 2021-11-08 at 2 20 53 PM](https://user-images.githubusercontent.com/3759828/140804412-576a2516-05b1-4643-ac82-90cbcdc89333.png)
 
They shouldn't have changed the interface tho... :/